src/components/TextInputFocusable/index.js

```diff
@@ -35,6 +35,9 @@ const propTypes = {
 
     // Callback to fire when a file is dropped on the text input
     onDrop: PropTypes.func,
+
+    // If the chatswitcher is shown in iOS safari browser, the input should be disabled
+    isDisable: PropTypes.bool,
```

Comment:
Let's call this `isDisabled` instead of `isDisable`

Reply:
Updated.